KronorComponents/Common/EmbeddedPaymentViewModel.swift

```diff
@@ -28,13 +29,16 @@ enum SupportedEmbeddedMethod {
             return "vipps"
         case .payPal:
             return "paypal"
+        case .bankTransfer:
+            return "bankTransfer"
         case .fallback(let name):
             return name
         }
     }
     
     func isFallback() -> Bool {
         switch self {
+        case .bankTransfer: return true
         case .fallback(_): return true
```

Comment:
We can do the same combined case syntax here:
`case .fallback, .bankTransfer: return true`